src/main/java/fr/iut/pathpilotapi/routes/dto/PositionDTO.java

```diff
@@ -40,6 +41,10 @@ public PositionDTO(double longitude, double latitude) {
         this.latitude = latitude;
     }
 
+    public static PositionDTO createFromSalesman(Salesman salesman) {
```

Comment:
add comments

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-API/pull/15/commits/5f466b6e7aeef0eb64b3740a9cbd407158d48d38